packages/polaris-nest/guides/README.md

```diff
@@ -44,20 +44,34 @@ export class AuthorModule {}
 ```typescript
 @Injectable({ scope: Scope.REQUEST })
 export class AuthorService {
+    private authorRepository: PolarisRepository<Author>;
     constructor(
-        @InjectRepository(Author)
-        private readonly authorRepository: PolarisRepository<Author>,
-        @InjectConnection()
-        private readonly connection: PolarisConnection,
         @Inject(CONTEXT) private readonly ctx: TestContext,
-    ) {}
+        @Inject(PolarisTypeORMInjector)
+        private readonly polarisTypeORMInjector: PolarisTypeORMInjector,
+    ) {
+        this.authorRepository = this.polarisTypeORMInjector.getRepository(Author);
+    }
 
     public async create(firstName: string, lastName: string): Promise<Author> {
         const author = new Author(firstName, lastName);
         return ((await this.authorRepository.save(this.ctx, author)) as unknown) as Promise<Author>;
```

Comment:
return ((await this.authorRepository.save(this.ctx, author)) as unknown) as Promise<Author>;
remove this.ctx